linux_pipeline/Jenkinsfile_lis_rpm_build_azure

```diff
@@ -1,5 +1,36 @@
 #!/usr/bin/env groovy
 
+def build_distro5 = 'no'
+def build_distro6 = 'no'
+def build_distro7 = 'no'
+def architecture = "x86_64"
+
+if(arch == "x32")
+        architecture = "i686"
```

Comment:
Have you verified that there is no need of curly braces? (I'm not sure)

Reply:
Yes